sink_service/source/main.c

```diff
@@ -20,10 +20,8 @@
 
 #define LOG_MODULE_NAME "Main"
 #define MAX_LOG_LEVEL INFO_LOG_LEVEL
-#include "logger.h"
 
-/* Default serial port */
-static char * port_name = "/dev/ttyACM0";
```

Comment:
So this part of the change is just changing some line position ;-)

Reply:
Oh oh you deserve a :cookie: for this one :+1: